lib/plug/adapters/test/conn.ex

```diff
@@ -8,6 +8,10 @@ defmodule Plug.Adapters.Test.Conn do
     maybe_flush()
 
     uri = URI.parse(uri)
+    if is_binary(uri.path) && not String.starts_with?(uri.path, "/") do
+      raise ArgumentError, message: "The URI path needs to start with \"/\", got #{uri.path}"
```

Comment:
Let me know if that error message is clear enough.

Reply:
The message is great! However:

1. You don't need to pass `message:`, by convention we do `raise ArgumentError, "..."`
2. We start error messages with lowercase letters
3. I would `inspect/1` the `uri.path` so that folks see the actual Elixir term they passed in.

This is what I'm talking about:

```suggestion
      raise ArgumentError, "the URI path needs to start with \"/\", got: #{inspect(uri.path)}"
```